streams/src/main/java/org/apache/kafka/streams/kstream/internals/KGroupedTableImpl.java

```diff
@@ -157,4 +172,18 @@ public Long apply(K aggKey, V value, Long aggregate) {
                 Serdes.Long(), storeName);
     }
 
+    private <T> StateStoreSupplier keyValueStore(final Serde<T> aggValueSerde, final String name) {
```

Comment:
This method needs to return `StateStoreSupplier<KeyValueStore>`. Also, it looks like this method and the next one are identical to methods with the same name in `KGroupedStreamImpl` - is there someway we can avoid the duplication?